crates/zizmor/src/main.rs

```diff
@@ -605,6 +605,7 @@ fn run() -> Result<ExitCode> {
         }};
     }
 
+    register_audit!(audit::anonymous_definition::AnonymousDefinition);
```

Comment:
Nitpick: mind putting this at the bottom of this group! Right now the order technically doesn't matter but by convention I've been ordering them by introduction time within the registry 🙂